UI/DesignWnd.cpp

```diff
@@ -159,15 +157,12 @@ namespace {
 
             for (const path& design_path : design_files) {
                 try {
-                    std::map<std::string, ShipDesign*> file_designs;
+                    std::map<std::string, std::unique_ptr<ShipDesign>> file_designs;
                     parse::ship_designs(design_path, file_designs);
 
-                    for (std::map<std::string, ShipDesign*>::value_type& design_entry : file_designs) {
+                    for (auto& design_entry : file_designs) {
```

Comment:
After reading a bit more (using more accredited sources), it does not seem to be significant in this case.  I failed to find any adverse effects, possibly providing better support for emplace/forward (though I may be wrong about this as well).